### PR TITLE
Generation of app json

### PR DESCRIPTION
**Pull request by matthiasfeist**

### CODE REVIEW

File: src/data/compile-app-json.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+function processSpeaker() {
+  var result = []
+
+  for (var speaker of SpeakerList) {
+    result.push({
+      name: speaker.name,
+      twitter: '',
+      about: speaker.bio,
+      location: 'Klarna',
+      profilePic: speaker.portrait,
```
Comment (mommel) [inline src/data/compile-app-json.js]: Speaker Images have to be
"profilePic": "assets/img/speakers/{NAME_HERE}.jpg",
and can't be the ones taken from web

File: src/data/compile-app-json.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+function processTeam() {
+  var result = []
+
+  for (var speaker of teamList) {
+    result.push({
+      name: speaker.name,
+      twitter: '',
+      about: speaker.bio,
+      location: 'Klarna',
+      profilePic: speaker.portrait,
```
Comment (mommel) [inline src/data/compile-app-json.js]: Same as the speakers but
"profilePic": "assets/img/team/{NAME}.jpg",

File: package.json [hunk 3]
```diff
@@ -52,8 +52,10 @@
     "start": "cross-env NODE_ENV=development webpack-dev-server --config configs/webpack.config.js",
     "test": "cross-env NODE_ENV=test nyc mocha-webpack --timeout=100000 --colors --webpack-config configs/webpack.config.test.js test/**/*.test.*",
     "build": "cross-env NODE_ENV=production webpack --config configs/webpack.config.js",
-    "clean": "rimraf dist .tmp .nyc_output",
-    "deploy": "npm run build && gh-pages -d dist"
+    "clean": "rimraf dist tmp .tmp .nyc_output",
+    "deploy": "npm run appjson && pm run build && gh-pages -d dist",
```
Comment (grozen) [inline package.json]: `pm run build` -> `npm run build`

File: src/data/compile-app-json.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  var result = []
+
+  for (var speaker of SpeakerList) {
+    result.push({
+      name: speaker.name,
+      twitter: '',
+      about: speaker.bio,
+      location: 'Klarna',
+      profilePic: speaker.portrait,
+      talkPic: speaker.portrait
```
Comment (mommel) [inline src/data/compile-app-json.js]: Talk Pictures are special pictures for the lunch,dinner,reg and for the speakers the konferense logo

**Comment by mommel** [comment]
Ok should work 👍

**Pull request merged**